### PR TITLE
Hire with Us Hero Component

### PR DESCRIPTION
**Pull request by mdPial**
This PR is for the issue #200 .

- [x] Things are made/changed
* `Route` for the page
* image in `public/static/images/hire-with-us/hero.png`
* `Hero` component **HeroWithBackgroundImage**
* In  `src/views/pages/HireWithUsView/Hero.js`
      * index page
      * Hero component

### CODE REVIEW

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Cool, 
Some minors to change:
- Lesson the padding of the content on smaller screens.
- make the button opaque on hover, opacity should be 1. We will likely change the color of the button on hover too, but lets try it before something else.

**Comment by mdPial** [comment]
@Abhishek-kumar09 Sorry, I didn't understand. What you meant by the `Lesson` word

**Comment by Abhishek-kumar09** [comment]
Oh Sorry, It is lessen meaning change padding to be little less than the usual desktop paddings when switched to smaller screens.

**Comment by mdPial** [comment]
Oh. No problem. I actually guessed that after thinking then I thought any keyword might be for professional works 🤣

**Comment by mdPial** [comment]
@Abhishek-kumar09 I got confused about reducing padding because I haven't used any 😅️. Though I have done some changes based on the screen. Please let me know if I made any mistakes. And please give me some guideline on asking questions maybe I am not asking questions properly because of my shyness.

File: src/components/Hero/HeroWithBackgroundImage.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    marginBottom: '3rem',
+    [theme.breakpoints.down('sm')]: {
+      marginTop: '1.3rem',
+      marginBottom: '1.5rem'
+    }
+  },
+  content: {
+    marginLeft: '150px',
+    [theme.breakpoints.down('sm')]: {
+      marginLeft: '120px'
```
[the remark this answers is not in the document]
Comment (mdPial) [reply src/components/Hero/HeroWithBackgroundImage.js:44]: [theme.breakpoints.down('sm')]: {
      margin: theme.spacing(3)
}
like this?

[the remark this answers is not in the document]
Comment (Abhishek-kumar09) [reply src/components/Hero/HeroWithBackgroundImage.js:44]: `[theme.breakpoints.down('sm')]: margin: theme.spacing(3) `
like this

[the remark this answers is not in the document]
Comment (mdPial) [reply src/components/Hero/HeroWithBackgroundImage.js:44]: Oh, Okay, github formatted my code into one line 😅️. I just wanted to ask. Do I have to give the margin all side like that or only in the left side.

**Comment by mdPial** [comment]
@Abhishek-kumar09 I have made changes.

**Comment by Abhishek-kumar09** [comment]
Ya, its looking really good. I will review the changes after some time, if it is ok.

**Comment by mdPial** [comment]
> Ya, its looking really good. I will review the changes after some time, if it is ok.

Take your time. I have no problem😃️

**Review by Abhishek-kumar09 — APPROVED** [review]

**Comment by Abhishek-kumar09** [comment]
Thanks @mdPial for your wonderful contribution.

**Comment by Abhishek-kumar09** [comment]
merging once ci pass

**Comment by mdPial** [comment]
> Thanks @mdPial for your wonderful contribution.

Welcome. Feeling great, at last merged 😁️

**Comment by Abhishek-kumar09** [comment]
You can also checkout click-short or certificate-generator-react they are also new projects that we are building now, but due some time constraints it is slow.

**Comment by mdPial** [comment]
> You can also checkout click-short or certificate-generator-react they are also new projects that we are building now, but due some time constraints it is slow or could also try to solve some new issues in this repo.

Thank you for the suggestion 😀️. I was looking to contribute as much as I can possible. Because I had gained the courage to contribute after watching videos of **codeforcause** 😁️. Thank you for guideline 😃️ along the way I never used Material UI before.

**Comment by Abhishek-kumar09** [comment]
Thanks for kind words.

**Pull request merged**